src/main/java/com/example/gradecalculator/entities/User.java

```diff
@@ -1,4 +1,61 @@
 package com.example.gradecalculator.entities;
 
+import jakarta.persistence.*;
+
+@Entity
 public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String name;
+    private String email;
+    private String message;
+
+    @ManyToOne
+    @JoinColumn(name = "user_type_id")
+    private UserType userType;
+
+    public User() {
+    }
+
+    public User(String name, String email, String message) {
+        this.name = name;
+        this.email = email;
+        this.message = message;
+    }
+
+    public User(String doe, String john) {
```

Comment:
also a constructor that has parameters but does never use those parameters

Reply:
I should've renamed it to "name" and "surname". It's fixed

Reply:
but values are never assigned. use empty constructor instead